README.md

```diff
@@ -190,7 +208,20 @@ See the [migration instructions](https://github.com/faceyspacey/redux-first-rout
 **TODO**
 
 #### Use Redux Devtools to debug route changes
-**TODO**
+`yarn add -D redux-devtools-extension`
+```diff
+// configureStore.js
+-import { createStore, applyMiddleware, compose, combineReducers } from 'redux'
++import { createStore, applyMiddleware, combineReducers } from 'redux'
+ import { connectRoutes } from 'redux-first-router'
++import { composeWithDevTools } from 'redux-devtools-extension'
+ 
+-  const enhancers = compose(enhancer, middlewares)
++  const enhancers = composeWithDevTools(enhancer, middlewares)
```

Comment:
there should be come `typeof window !== 'undefined'
```suggestion
+  const toCompose = typeof window !== 'undefined' ? composeWithDevTools : compose
+  const enhancers = toCompose(enhancer, middlewares)
```

Reply:
that way devtools dont leak into prod

Reply:
You're right, I guess I thought the library would handle that under the hood. I'll try to fix this later today.

Reply:
Thanks man :) after that, ill marge